### PR TITLE
Many to many table

### PR DESCRIPTION
**Pull request by pinkmoon25**
### Add Join table for visits and implement many to many relationship

- Create Vets, Specialisations and Visits tables
- Implement many to many relation between visits and vets and between specialization and vets
- Used primary and foreign key for joining tables
- Write complex queries to answer multiple questions

query3

query_unspecialised_visit

query5

qury4

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -12,13 +12,31 @@ CREATE TABLE animals(
...
+
+create table specializations (
+    vet_id int references vets(id),
+    species_id int references species(id)
+);
+
+create table visits (
+vet_id int references vets(id),
+animals_id int references animals(id)
+);
```
Comment (cynthiainga) [inline schema.sql]: - [x] Good job so far by implementing the `visits` table, but it should also keep track of the date of the visit as required in the project 🙃, please kindly add a column of visit date 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @PinkMoon25, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Used correct gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (pinkmoon25) [reply schema.sql]: Thank you @cynthiainga , I have added date of visit to the visits table

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @PinkMoon25,

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**